.github/workflows/tests.yml

```diff
@@ -18,18 +18,26 @@ jobs:
     strategy:
       matrix:
         include:
-          - php-version: 7.2
+          - php-version: 7.4
             symfony-version: 5.4.*
           - php-version: 8.1
             symfony-version: 5.4.*
           - php-version: 8.1
             symfony-version: 6.4.*
           - php-version: 8.3
             symfony-version: 6.4.*
+          - php-version: 8.4
```

Comment:
I don't want to test ALL the PHP + Symfony combinations
Usually I do 2 PHP version for each Symfony version: lowest and highest
I try to keep only 1 Symfony version for each major: the highest available

So:
- Symfony 6.4.*
  - PHP 8.1
  - PHP 8.4
- Symfony 7.2.*
  - PHP 8.1
  - PHP 8.4

Reply:
Done, i also change it for the phpstan and checkstyle jobs!